.github/workflows/test_production_build.yml

```diff
@@ -0,0 +1,78 @@
+name: Test production build
+
+on: push
+
+env:
+  APP_NAME: rails_templates
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  RUBY_VERSION: 2.7.1
+  NODE_VERSION: 12
```

Comment:
@hoangmirs Node 14 become current LTS version so it's ok to upgrade to NODE_VERSION 14, what do you think?